docs/devel/cherry-picks.md

```diff
@@ -47,6 +47,20 @@ hack/cherry_pick_pull.sh upstream/release-3.14 98765
 This will walk you through the steps to propose an automated cherry pick of pull
  #98765 for remote branch `upstream/release-3.14`.
 
+### Cherrypicking a doc change
+
+If you are cherrypicking a change which adds a doc, then you also need to run
+build/versionize-docs.sh in the release branch to versionize that doc.
+Ideally, just running cherry_pick_pull.sh should be enough, but we are not there
+yet: https://github.com/kubernetes/kubernetes/issues/18861
```

Comment:
Nit: prefer the [#18861](https://github.com/kubernetes/kubernetes/issues/18861) syntax.